CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Expose port 8888 for VerneMQ metrics.
 - Added a Helm chart for deploying the Operator
 - Added Kubernetes Webhooks
+- Allow disabling webhooks with ENABLE_WEBHOOKS env var.
```

Comment:
Does it make sense allowing this in Helm, too? If so, the cert-manager dependency could be avoided in those cases

Reply:
In the long short run it should be ok. However if we want to rely on cert-manager on a wider scope, I wouldn't make the dependency optional.